tendrl/ceph_integration/manager/request_factory.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, cluster_monitor):
     def delete(self, obj_id):
         raise NotImplementedError()
 
+    def delete_rbd(self, pool_id, name):
```

Comment:
Had tried existing `delete` but that take only id to be deleted. there is no way we can pass `pool_id` and `rbd_name` both